### PR TITLE
Choice parsing: support turn timeout decisions

### PR DESCRIPTION
**Pull request by Slayer95**

### CODE REVIEW

**Comment by Zarel** [comment]
...are your commit descriptions more than 72 chars wide? o_O

**Comment by Zarel** [comment]
I'm not sure if this is really the way I'd implement it, but this is better than the current system.

**Comment by Slayer95** [comment]
Inserting random line breaks is a burden. Do we care about that? I've only limited commit headers so far...

How would you implement it? I thought on the alternative of having a <code>pokemon.resolveDecision()</code> method and call it at <code>Battle.choose</code> for <code>'default'</code> choices, but that increases the times <code>'LockMove'</code> and <code>'LockMoveTarget'</code> are run.

File: battle-engine.js [hunk 1]
```diff
@@ -3899,7 +4009,7 @@ Battle = (function () {
 			case 'pass':
 				if (i > side.active.length || i > side.pokemon.length) continue;
 				if (side.currentRequest !== 'switch') {
-					this.debug("No se pudo pasar el turno.");
+					this.debug("Can't pass the turn");
```
Comment (Zarel) [inline battle-engine.js]: lol

**Comment by Zarel** [comment]
I limit my own commit messages to 72 columns because that's what I've been told, and because it plays better with certain commandline Git commands.

I don't use any of those commands personally, though, so I don't care too much if you leave off the manual wrapping.

**Comment by Zarel** [comment]
I'm not sure how exactly I'd personally implement this (if I cared enough to think through how to refactor it, I'd just commit said refactor), but intuitively this seems like more code than necessary.

File: battle-engine.js (hunk 1 above)
Comment (DanUgore) [inline battle-engine.js]: C'mon man this was Joim's legacy :(

Comment (Slayer95) [inline battle-engine.js]: It was mine pls

**Pull request merged**